api/src/server.ts

```diff
@@ -21,22 +21,24 @@ export default async function createServer() {
 			const elapsedNanoseconds = elapsedTime[0] * 1e9 + elapsedTime[1];
 			const elapsedMilliseconds = elapsedNanoseconds / 1e6;
 
-			const previousIn = (req.connection as any)._metrics?.in || 0;
-			const previousOut = (req.connection as any)._metrics?.out || 0;
```

Comment:
Is `socket` truly a drop-in replacement for connection?

Reply:
Yep, it looks like it was just renamed.